src/templates/initial-setup-script.sh.tpl

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+CicdNamespace="${CICD_NAMESPACE:-{{ .CicdNamespace }}}"
+
+# functions
+function getNotSynced() {
+  oc get applications --insecure-skip-tls-verify -o jsonpath='{.items[*].status.sync}' --kubeconfig="${KUBECONFIG}" -n openshift-gitops | jq 'select(.status != "Synced")' -c | wc -l
```

Comment:
namespace openshift-gitops ersetzen mit "gepardec-run-gitops" - kommt noch häufiger vor in dem skript